word-guess-main.rb

```diff
@@ -0,0 +1,86 @@
+
+
+class Game
+
+  def initialize(input_word, word_descriptions)
+    @word = input_word.chars.to_a
+    @hidden_word = @word.join.gsub(/[A-Za-z]/, "_")
+    @bad_guesses = 0
+    @word_descriptions = word_descriptions
+  end
+
+  def guess(letter)
+    if @word.include?(letter) == false
+      @bad_guesses += 1
+      puts "I'm sorry, that letter is not in this word!"
+    else
+      puts "Excellent choice!!"
+      until @word.include?(letter) == false
+        index = @word.index(letter)
+        @word[index] = "*"
+        @hidden_word[index] = letter
+      end
+    end
+  end
+
+  def word_descriptions
+    return "#{@word_descriptions}"
+  end
```

Comment:
Since `@word_descriptions` is already a string, I think you could use `attr_reader :word_descriptions` with the same effect.